digitalaidseattle/seattle-humane-app
# Feature/5.1.1.1/search query

## What type of PR is this? (check all applicable)


## Description

Adds a search field to the dashboard. Users can search across all the fields that appear in the tables on the dashboard.
Submitting a search runs a DB full text search. The results of the search is returned to the useAllTickets hook, so all the tables get the search results, and then they apply their client-side filtering (my cases, closed cases, etc.) on top of the search results.

## Related Tickets & Documents

- Closes # [5.3.1.3 Search Features](https://projects.digitalaidseattle.org/project-tasks-details?recordId=reco4WxwaGUHMElQt)

## QA Instructions, Screenshots, Recordings

1. Load the home page/dashboard. 
2. Enter a search term (or multiple, separated by spaces) into the search bar.
3. Press Enter or click the search button.
4. Verify the results in the tables match the query
5. Click the clear button or delete the query and press Enter
6. Verify the tables return to showing the original records

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_
- [ ] Color contrast tested? _more info [here](https://webaim.org/articles/contrast/evaluating)_

## Added/updated tests?
_There should be 90% statement coverage and the coverage percent increases with each commit, unless a reviewer approves an exception._

      have not been included_

## Can reviewers merge this PR upon approval?
- [ ] yes, reviewers can merge this PR
- [x] no, the author of this PR needs to merge it

## [optional] Are there any post-deployment tasks we need to perform?

## [optional] Which Seattle Humane animal would you like to dedicate this PR to?
# [Marinara](https://www.seattlehumane.org/?pet=56590391) 🥹
![Marinara](https://g.petango.com/photos/337/5f3ed5c2-234f-42bf-9f76-414b1f52f673.jpg)
